src/components/ReportActionItem/TaskPreview.tsx

```diff
@@ -24,141 +21,135 @@ import * as LocalePhoneNumber from '@libs/LocalePhoneNumber';
     }
 
     return (
         <View style={[styles.chatItemMessage]}>
             <PressableWithoutFeedback
-                onPress={() => Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(props.taskReportID))}
+                onPress={() => Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(taskReportID))}
                 onPressIn={() => DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
                 onPressOut={() => ControlSelection.unblock()}
-                onLongPress={(event) => showContextMenuForReport(event, props.contextMenuAnchor, props.chatReportID, props.action, props.checkIfContextMenuActive)}
+                onLongPress={(event) => showContextMenuForReport(event, contextMenuAnchor, chatReportID, action ?? {}, checkIfContextMenuActive)}
                 style={[styles.flexRow, styles.justifyContentBetween]}
                 role={CONST.ROLE.BUTTON}
-                accessibilityLabel={props.translate('task.task')}
+                accessibilityLabel={translate('task.task')}
             >
                 <View style={[styles.flex1, styles.flexRow, styles.alignItemsStart]}>
                     <Checkbox
                         style={[styles.mr2]}
                         containerStyle={[styles.taskCheckbox]}
                         isChecked={isTaskCompleted}
-                        disabled={!Task.canModifyTask(props.taskReport, props.currentUserPersonalDetails.accountID, lodashGet(props.rootParentReportpolicy, 'role', ''))}
+                        disabled={!Task.canModifyTask(taskReport ?? {}, currentUserPersonalDetails.accountID, rootParentReportpolicy ? rootParentReportpolicy.role : '')}
                         onPress={Session.checkIfActionIsAllowed(() => {
                             if (isTaskCompleted) {
-                                Task.reopenTask(props.taskReport);
+                                Task.reopenTask(taskReport ?? {});
                             } else {
-                                Task.completeTask(props.taskReport);
+                                Task.completeTask(taskReport ?? {});
                             }
                         })}
-                        accessibilityLabel={props.translate('task.task')}
+                        accessibilityLabel={translate('task.task')}
                     />
                     <RenderHTML html={htmlForTaskPreview} />
                 </View>
                 <Icon
                     src={Expensicons.ArrowRight}
-                    fill={StyleUtils.getIconFillColor(getButtonState(props.isHovered))}
+                    fill={StyleUtils.getIconFillColor(getButtonState(isHovered))}
                 />
             </PressableWithoutFeedback>
         </View>
     );
 }
 
-TaskPreview.propTypes = propTypes;
-TaskPreview.defaultProps = defaultProps;
 TaskPreview.displayName = 'TaskPreview';
 
 export default compose(
```

Comment:
Please refer to TS guidelines, unfortunately we have to remove `compose` here, as it isn't working as expected. [link](https://github.com/zhenjahorbach/app/blob/dcdf95b12e86c34f0efb4f366c0b01ee02fabf0b/contributingGuides/TS_STYLE.md#compose-usage)

Reply:
Just updated.